spec/models/services/facebook_spec.rb

```diff
@@ -22,39 +22,57 @@
       post_params = @service.create_post_params(post)
     end
 
-    it 'does not add post link when no photos' do
+    it "adds (via <post URL>) when message has no URLs " do
       message = "Some text."
       post = double(message: double(plain_text_without_markdown: message, urls: []), photos: [])
       post_params = @service.create_post_params(post)
-      expect(post_params[:message]).not_to include "http"
+      expect(post_params[:message]).to include "(via http:"
     end
 
-    it 'sets facebook id on post' do
-      stub_request(:post, "https://graph.facebook.com/me/feed").
-	to_return(:status => 200, :body => '{"id": "12345"}', :headers => {})
+    it "adds (via <post URL>) when message has URLs" do
```

Comment:
This test is only a duplicate of the test above, because the fact that there is an URL doesn't make a difference here. So you can just remove this and remove the "when message has no URLs " from the test above.

Reply:
OK. I like having this to make sure that if someone changes this later it doesn't get hosed up with embedded URLs but I'm fine making these changes.

Reply:
If somebody changes it later, they should add a test for the changed behavior. But if somebody changes it later so it doesn't contain the "via <post URL>" when it contains a mention (just a random example which makes no sense), then you don't have a test for it. And you shouldn't add test for every random case somebody maybe could do (which doesn't make a difference now). If it makes sense to change something in the future, then we can add a test for that later (and then this test maybe should fail and will be removed).